src/styles/nav.scss

```diff
@@ -0,0 +1,130 @@
+html {
+  margin: 0;
+}
+
+.catalog {
+  width: 1024px;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  padding: 50px 40px;
+  &__card {
+    margin-right: 48px;
+    margin-bottom: 46px;
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+    &:nth-last-child(-n + 4) {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.card {
+  width: 200px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: $white-color;
+  box-sizing: border-box;
+
+  &_cards {
+    margin-right: 48px;
+    margin-bottom: 46px;
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $text-accent-color;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    padding-bottom: 16px;
+  }
+
+  &__review-container {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    align-items: flex-end;
+    text-align: right;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review-count {
+    color: $text-accent-color;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    color: $text-accent-color;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    height: 40px;
+    border: 1px solid $main-color;
+    border-radius: 5px;
+    font-size: 14px;
+    line-height: 16px;
+    color: $white-color;
+    font-weight: 700;
+    text-decoration: none;
+    text-transform: uppercase;
+    background-color: $main-color;
+  }
+
+  &__buy:hover {
+    color: $main-color;
+    background-color: $white-color;
+  }
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background: url(../images/star-active.svg) no-repeat 50%;
+}
```

Comment:
add active stars for all `1, 2, 3, 4, 5` counts

Reply:
not fixed